core/src/mast/mod.rs

```diff
@@ -48,18 +49,18 @@ mod tests;
 ///
 /// A [`MastForest`] does not have an entrypoint, and hence is not executable. A [`crate::Program`]
 /// can be built from a [`MastForest`] to specify an entrypoint.
-#[derive(Clone, Debug, Default, PartialEq, Eq)]
```

Comment:
Is this a local lint you have? If so it would be nice if this could be done with `rustfmt`

Reply:
Just a mistake in this case.